identity-iota/src/error.rs

```diff
@@ -48,4 +46,13 @@ pub enum Error {
   CompressionError,
   #[error("invalid message flags")]
   InvalidMessageFlags,
+  /// Caused by a single validation unit failing.
+  #[error("A validation unit failed")]
+  UnsuccessfulValidationUnit(#[from] crate::credential::errors::ValidationError),
+  /// Caused by a failure to validate a credential.  
+  #[error("credential validation failed")]
+  UnsuccessfulCredentialValidation(#[source] crate::credential::errors::AccumulatedCredentialValidationError),
+  /// Caused by a failure to validate a presentation.
+  #[error("presentation validation failed")]
+  UnsuccessfulPresentationValidation(#[source] crate::credential::errors::AccumulatedPresentationValidationError),
```

Comment:
Nit: can we use `*Error` for consistency?

```suggestion
  /// Caused by a single validation unit failing.
  #[error("A validation unit failed")]
  ValidationError(#[from] crate::credential::errors::ValidationError),
  /// Caused by a failure to validate a credential.  
  #[error("credential validation failed")]
  CredentialValidationError(#[source] crate::credential::errors::AccumulatedCredentialValidationError),
  /// Caused by a failure to validate a presentation.
  #[error("presentation validation failed")]
  PresentationValidationError(#[source] crate::credential::errors::AccumulatedPresentationValidationError),
```